src/rules/russian.toml

```diff
@@ -0,0 +1,118 @@
+min_trimmed_length = 3
+
+# For count = 2 we have sentences like: "Это он", "Будет беда"
+# and a lot of trash abbreviations like "до н.", "и доп."
+min_word_count = 3
+
+max_word_count = 14
+
+# In Russian, words can consist of one letter, so no restrictions
+min_characters = 0
+
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+
+# Apparently, in some places the sentences are cut incorrectly,
+# which is why we get some part of the sentence, and not its entirety.
+# This is required to fix sentences like:
+# с Иваном Галамяном.
+needs_uppercase_start = true
+
+# The following are symbols that are either absent in the Russian language
+# or participating in language constructs that will be difficult or impossible to voice,
+# as well as constructions causing problems in sentences
+# (for example, opening quotes without closing)
+disallowed_symbols = [
+  '<', '>', '+', '*', '\', '#', '@', '^', '[', ']', '(', ')', '/',
+  'é', 'è', 'à', 'ç', 'Å', '薛', '⋅', '·', 'ѿ', '=', '|', '≡', '→',
+  '×', 'Ա', 'բ', 'դ', 'ի', 'ս', 'յ', 'շ', 'ո', '•', 'थ्', '©', '¸',
+  '天', '不', '言', '，', '以', '行', '與', '事', '示', '之', '而',
+  '已', '矣', '《', '大', '学', '集', '注', '》', '多', '识', '录',
+  '「', '鴃', '」', '鳩', '鳧', '鶯', 'ا', 'ل', 'ر', 'د', 'ن',
+  'เ', 'วฺ', 'ส', 'ลฺ', 'ยุ', 'ทิ', 'โ', 'ร', 'ชฺ', 'ท', 'ยุ', 'ตฺ', 'สฺ',
+  'ย', 'พ', 'ทฺ', 'นิ', 'इ', '_', 'Ѳ', '‹', '›', '`',
+  'і', 'ў', '~', '®', '%', '⟨', '⟩', '¬', '&', '≤',
+  'α', 'β', 'Γ', 'γ', 'Δ', 'δ', 'ε', 'ζ', 'η', 'Θ', 'θ', 'ι', 'Ә',
+  'Λ', 'λ', 'μ', 'ν', 'Ξ', 'ξ', 'ρ', 'Σ', 'σ', 'ς', 'τ',
+  'υ', 'Φ', 'φ', 'χ', 'Ψ', 'ψ', 'Ω', 'ω', 'a', 'A', 'b', 'B', 'c',
+  'C', 'd', 'D', 'e', 'E', 'f', 'F', 'g', 'G', 'h', 'H', 'i', 'I',
+  'j', 'J', 'k', 'K', 'l', 'L', 'm', 'M', 'n', 'N', 'o', 'O', 'p',
+  'P', 'q', 'Q', 'r', 'R', 's', 'S', 't', 'T', 'u', 'U', 'v', 'V',
+  'w', 'W', 'x', 'X', 'y', 'Y', 'z', 'Z',
+  'җ', 'ә', 'ү', 'ː', 'қ', 'ћ', 'Ä', 'Ҋ', 'ј', 'Ι', 'Χ',
+  
+  # There is a lot of broken sentences like:
+  # У меня щемило сердце: «Что мы сделаем?
+  # Такой он был веселый, бодрый, нас воодушевлял: „Вы знаете, не бойтесь!
+  # История и культура»".
+  # Пушкин, прочитав поэму, сказал «— Далеко мальчик пойдет».
+  # После увольнения он пытается создать новую жизнь...Жизнь, в свою очередь, строит ему множество «баррикад"
+  # Пушкин, Набоков… мало кого еще можно поставить в этот ряд?
+  # Let's remove them all, we have so many good sentences without problems
+  '«', '»', '"', '„', '“', '”', '‘', '…',
+  '{', '}', '（', '）',
```

Comment:
Even if we have enough sentences when filtering out all of them, there is the option `even_symbols` (https://github.com/Common-Voice/common-voice-wiki-scraper/blob/master/README.md#using-language-rules). I've just added a simplification to the master branch, so you might want to update your branch. Then you can add all these into the `even_symbols` array and the script will make sure to only take sentences with even count of these symbols.